src/App.js

```diff
@@ -1,43 +1,25 @@
-import "./App.css";
-import logo from "./logo.png";
+import "./styles.css";
+import { Routes, Route } from "react-router-dom";
+import Header from "./component/header";
```

Comment:
You can import all the components from a single file created as an index and import all the components at once and export from their, and you can save your space in other files